File: 800/Java/P9.java
import java.util.*;

public class P9 {
    static void solve(Scanner scanner) {
        int n = scanner.nextInt();
        int ans = 0;
        for(int i = 0; i < n-1; i++)    {
            int num = scanner.nextInt();
            ans -= num;
        }

        System.out.println(ans);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        while(t-- > 0)  solve(scanner);
        scanner.close();
    }
}

File: 800/C++/p9.cpp
#include<bits/stdc++.h>
using namespace std;

void solve() {
    int n;
    cin>>n;

    int ans = 0;
    for(int i = 0; i < n-1; i++) {
        int num;
        cin>>num;
        ans -= num;
    }

    cout<<ans<<endl;
}

int main() {
    int t;
    cin>>t;
    while(t--)  solve();

    return 0;
}